profiles/base10/modules/config.tpl.xqm

```diff
@@ -457,26 +460,28 @@ declare variable $config:dts-import-collection := $config:data-default || "/play
  : @param $docUri relative document path (including $collection)
  :)
 declare function config:collection-config($collection as xs:string?, $docUri as xs:string?) {
-    (: Return empty sequence to use default config :)
-    ()
-
-    (: 
-     : Replace line above with the following code to switch between different view configurations per collection.
-     : $collection corresponds to the relative collection path (i.e. after $config:data-root). 
-     :)
-    (:
-    switch ($collection)
-        case "playground" return
+    [% if exists($context?collection-config) %]
```

Comment:
I started to move templated sections into `generated-config.tpl.xql` to allow users to change `config.xqm` manually without running into a conflict on next update.

Reply:
I fixed it, thanks!